.deepsource.toml

```diff
@@ -0,0 +1,19 @@
+version = 1
+
+test_patterns = [
+    "tests/**",
+    "**/test_*.py"
+]
+
+exclude_patterns = [
+    "examples/**",
+    "helper/**",
+    "menubar/**"
+]
```

Comment:
Can we also include a node/typescript analyzer for `dashboard` directory?  It's not yet part of the release but no reason skip code quality checks.

Reply:
Sure, I'll add the TS analyzer too. Though it's in beta, and any active feedback on it would be appreciated :)